packer/openstack.pkr.hcl

```diff
@@ -109,7 +109,7 @@ variable "manifest_output_path" {
 
 variable "use_blockstorage_volume" {
   type = bool
-  default = true
+  default = false
```

Comment:
If we're doing this, can you modify docs/image-build.md b/c it currently says vol backed is default. Or, you could just set it to false in CI.